src/includes/configuration/sourcemaps/tools/webpack/advanced-usage/javascript.nextjs.mdx

```diff
@@ -0,0 +1,26 @@
+```javascript {filename:next.config.js}
+const { withSentryConfig } = require("@sentry/nextjs");
+
+const moduleExports = {
+  webpack: (config, options) => {
+    devtool = 'source-map',
+    config.output = {
+      ...config.output,
+      // Make maps auto-detectable by sentry-cli
+      filename: "[name].js",
+      sourceMapFilename: "[name].js.map",
+    };
+    return config;
+  },
+  productionBrowserSourceMaps: true,
```

Comment:
```suggestion
```

Since I made this suggestion,  I discovered that [we don't need this option at all](https://github.com/getsentry/sentry-javascript/pull/3765).